.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: release
+on:
+  workflow_call:
+    inputs:
+      build:
+        description: "Machine we are releasing for"
+        required: true
+        type: string
+      app_name:
+        description: "Machine we are releasing for"
```

Comment:
these descriptions probably should be updated to be accurate, obviously not an issue right this moment